content/en/blog/extend-falco-inputs-with-a-plugin-register.md

```diff
@@ -0,0 +1,112 @@
+---
+Title: "Extend Falco inputs by creating a Plugin: Register the plugin"
+Date: 2022-02-20
+Author: Thomas Labarussias
+slug: extend-falco-inputs-with-a-plugin-register
+---
+
+> This post is is part of a series of articles about `How to develop Falco plugins`. It's addressed to anybody who would like to understand how plugins are written and want to contribute.
+> See other articles:
+> * [Extend Falco inputs by creating a Plugin: the basics]({{< ref "/blog/extend-falco-inputs-with-a-plugin-the-basics" >}})
+
+- [Introduction](#introduction)
+- [The registry](#the-registry)
+- [Details of your plugin](#details-of-your-plugin)
+	- [License](#license)
+	- [ID](#id)
+	- [Name](#name)
+	- [Fields](#fields)
+- [Propose your Plugin](#propose-your-plugin)
+- [Share with the community](#share-with-the-community)
+- [Conclusion](#conclusion)
+	- [You should now be able to propose your plugin to the community and have feedback about it. The following posts will describe more advanced use-cases with events from Cloud Services. Stay tuned!](#you-should-now-be-able-to-propose-your-plugin-to-the-community-and-have-feedback-about-it-the-following-posts-will-describe-more-advanced-use-cases-with-events-from-cloud-services-stay-tuned)
+
+# Introduction
```

Comment:
I think it would be nice to expand this section a bit, what kind of things we cover, why we need to register the plugin, what "register" means, etc.